package.json

```diff
@@ -129,6 +130,11 @@
     "test:php": "phpunit",
     "test:php:help": "npm run test:php -- --help"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint"
+    }
+  },
```

Comment:
Maybe this is out of scope for this PR. It [doesn't overwrite](https://www.npmjs.com/package/husky) existing pre-commit hooks. If this is reverted, `package-lock.json` should also be reverted.